ffi_common/tests/no_compile/no_memberwise_init.rs

```diff
@@ -0,0 +1,14 @@
+use ffi_common::derive::FFI;
+
+#[derive(Debug, Clone, Copy, FFI)]
+#[ffi(forbid_memberwise_init)]
+pub struct NoInitializerAllowed {
+    pub field1: u8,
+}
+
+fn test_foo() {
```

Comment:
should this have a:

```rust
#[test]
#[should_panic]
```

or is the test handled by trybuild?

Reply:
No, it doesn't panic or test anything because this module can't even compile (`no_initializer_allowed_rust_ffi_init` doesn't exist, so when `trybuild` goes to run this, it gets the expected compiler error). This could definitely use a better name like `call_memberwise_init`, though.